test/e2e/framework/statefulset_utils.go

```diff
@@ -498,6 +496,7 @@ func NewStatefulSetPVC(name string) v1.PersistentVolumeClaim {
 					v1.ResourceStorage: *resource.NewQuantity(1, resource.BinarySI),
 				},
 			},
+			StorageClassName: &classAnything,
 		},
```

Comment:
But for this `anything` class to work - unlike alpha annotations this `anything` class must exist in the system. Did something create this class? If nobody created that class, it won't work.